packages/manager/apps/sign-up/src/index.js

```diff
@@ -74,11 +79,8 @@ angular
         window.navigator.userLanguage ||
         'en';
 
-      const userLocale = TranslateServiceProvider.findLanguage(
-        language,
-        language,
-      );
-      TranslateServiceProvider.setUserLocale(userLocale);
+      const userLocale = findLanguage(language, language);
```

Comment:
for clarity you should make country parameter optional in ```findLanguage``` method with a default value equals to language parameter, so you can simply call ```findLanguage(language)``` which i think is a lot better than ```findLanguage(language, language)``` ^^

Reply:
Fixed
`Environment.findLanguage('fr') === Environment.findLanguage('fr', 'fr')`